### PR TITLE
Change Manifest.json to reflect MakeSense Name

### PR DESCRIPTION
**Pull request by rasyidf**
I've noticed when trying to install as app via MS Edge. 😄

### Pre-flight checklist

- [x] Unit tests for all non-trivial changes
- [x] Tested locally
- [x] Updated wiki

### CODE REVIEW

**Comment by SkalskiP** [comment]
Haha 😅  That was unexpected.

File: public/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "MakeSense",
```
Comment (SkalskiP) [inline public/manifest.json:2]: makesense.ai

File: public/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "name": "MakeSense.ai Image Annotation",
```
Comment (SkalskiP) [inline public/manifest.json:3]: makesense.ai - free to use online tool for labelling photos

File: public/manifest.json (hunk 1 above)
Comment (rasyidf) [reply public/manifest.json]: Edited, please review

File: public/manifest.json (hunk 2 above)
Comment (rasyidf) [reply public/manifest.json]: I've edited accordingly.

**Comment by SkalskiP** [comment]
Thank you.

**Review by SkalskiP — APPROVED** [review]

**Pull request merged**